walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -139,15 +140,16 @@
   */
   private LinkedList<Event> removeEventsWithNoAttendees(HashSet<String> attendees, LinkedList<Event> events) {
     LinkedList<Event> sequentialEvents = new LinkedList<>(events);
+    ListIterator listIterator = sequentialEvents.listIterator();
```

Comment:
I think you can declare this a `ListIterator<Event>` listIterator to avoid having to do the `(Event)` cast below.

Also, despite what I said earlier about it being weird not having an increment condition in a for-loop, I'd say in this case it's probably better to use a for-loop just to keep the `listIterator` var scoped to the for-loop:
`for (ListIterator<Event> listIterator; listIterator.hasNext();)`